CHANGELOG.md

```diff
@@ -27,6 +27,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   * `templates.log_node`
   * `templates.op_log_node`
 
+* A default configuration for `templates.log_node` is provided out of the box,
+  which now colors and marks specific nodes in the graph for ease of reading.
+  
+  * Immutable nodes (as specified by the `immutable_heads()` revset) are
+    represented with blue diamonds `◆`
+  * Mutable commits are represented with ordinary uncolored circles `○`
+  * "Elided nodes" are specified with the "harpoon" symbol `⇋` to show that
+    there are commits between the two nodes that are not shown.
```

Comment:
Do we want to say something like `to show part of the line has been "cut" out`? That's what I assumed was the thinking behind the harpoons.

Reply:
ack